app/Views/movies/foot.php

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
You are calling this "foot", but it looks like an entire HTML page rather than just the bottom portion?